src/constants.ts

```diff
@@ -1,6 +1,6 @@
 // constants
 class Constants {
-    public static readonly adoc: string = ".asciidoc";
+    public static readonly adoc: string = ".{.asciidoc,adoc}";
```

Comment:
Just looking at the change, there is one dot too much within the brackets, isn't it?